<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImagePreviewer</class>
 <widget class="QWidget" name="ImagePreviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QToolButton{
background-color: rgba(255, 255, 255, 0);
}</string>
  </property>
  <widget class="QToolButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>336</y>
     <width>96</width>
     <height>96</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>next</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons8-right-96.png</normaloff>:/icons/icons8-right-96.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>96</width>
     <height>96</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="front_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>336</y>
     <width>96</width>
     <height>96</height>
    </rect>
   </property>
   <property name="text">
    <string>front</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons8-left-96.png</normaloff>:/icons/icons8-left-96.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>96</width>
     <height>96</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="show">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imageLB">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>70</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(158, 158, 158);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>No 
Image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="quitBtn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>650</y>
     <width>201</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="fileNameLB">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>710</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans Mono CJK TC</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color: rgba(0, 0, 0, 60);
border-radius: 10px;

color:white;
}
</string>
   </property>
   <property name="text">
    <string>FileName</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>imageLB</zorder>
  <zorder>show</zorder>
  <zorder>next_btn</zorder>
  <zorder>front_btn</zorder>
  <zorder>quitBtn</zorder>
  <zorder>fileNameLB</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
